odl_video/settings.py

```diff
@@ -234,6 +234,7 @@
 # Logging configuration
 LOG_LEVEL = get_string('ODL_VIDEO_LOG_LEVEL', 'INFO')
 DJANGO_LOG_LEVEL = get_string('DJANGO_LOG_LEVEL', 'INFO')
+LOG_FILE = get_string('ODL_VIDEO_LOG_FILE', None)
```

Comment:
This should be added to `app.json`

Reply:
It doesn't make sense to add it to app.json because we wouldn't ever want to use a log file on Heroku. This setting is only useful on EC2.